Composer/packages/client/src/store/persistence/FilePersistence.ts

```diff
@@ -28,6 +29,14 @@ class FilePersistence {
   private _files: { [fileName: string]: FileOperation };
   private _projectId = '';
   private _handleError = name => error => {};
+  private _queue = new FileQueue();
+  private _lock = false;
```

Comment:
this is not a `lock`, this is just a "isRunning" or "isSaveing"  flag if there is a saving function runnning.

Reply:
use a more accurate name would be better. lock is confusing

Reply:
isFlushing is the best i can think